src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: string;
+  export default content;
+}
```

Comment:
What is this doing?

Reply:
module so that typescript recognizes `*.svg` files

Reply:
Could you elaborate further? Why does TypeScript need to recognize `.svg` files?

Reply:
in `src/components/Header/index.stories.tsx` i'm import `cbi-logo.svg`, if I don't do this, it throws me this error about not finding the "module".

<img width="850" alt="image" src="https://user-images.githubusercontent.com/69323251/160488396-403c4b7a-2578-40c9-b74c-cdcb98ec5beb.png">

i tried to find an alternative to this, maybe something in `tsconfig.json` but i couldn't find anything, if you anybody knows of one, please let me know.

Reply:
Got it. Upon searching, this seems pretty standard. Works for me!